bin/kokkos_launch_compiler

```diff
@@ -65,6 +65,9 @@ CXX_COMPILER=${1}
 # remove the expected C++ compiler from the arguments
 shift
 
+# escape $
+target=("${@/\$/\\\$}")
```

Comment:
Why did you call this "target"?

Reply:
As in "target (command) for the script". Looking at that now, maybe `command` is better :thinking: since that's also what the comments in the script refer to.